subprojects/platform-play/src/main/java/org/gradle/play/plugins/PlayDistributionPlugin.java

```diff
@@ -168,6 +170,22 @@ public void execute(CreateStartScripts createStartScripts) {
         }
     }
 
+    private String getMainClass(PlayDistribution distribution) {
+        PlayPlatform playPlatform = distribution.getBinary().getTargetPlatform();
+        String playVersion = playPlatform.getPlayVersion();
+        switch (PlayMajorVersion.forPlatform(playPlatform)) {
+            case PLAY_2_2_X:
+            case PLAY_2_3_X:
+            case PLAY_2_4_X:
+            case PLAY_2_5_X:
+                return "play.core.server.NettyServer";
+            case PLAY_2_6_X:
+                return "play.core.server.ProdServerStart";
+            default:
+                throw new RuntimeException("Could not create Twirl compile spec for Play version: " + playVersion);
```

Comment:
This should probably say something like, `"Could not determine main class for Play version: " + playVersion`

Reply:
Bad copy and pasta. :-P

Will fix that too.